src/views/projectsView/ProjectFilters.jsx

```diff
@@ -51,6 +53,7 @@ function formatSearchPath(currentFilters) {
 }
 
 export default function ProjectFilters(props) {
+  const isTabletOrMobile = useMediaQuery({ query: "(max-width: 996px)" });
```

Comment:
Adding this comment after the fact, but one thing that would be good to consider for this project is to put values like `996px` that are likely to be reused in the future into a constants file. These types of values are often referred to as "magic numbers" and are considered a software anti-pattern. Magic numbers are hard-coded values that appear in code and lack context. Often, its better to assign such values a name and reuse that named value in multiple places. That way, if a breakpoint changes in the future, its easy to update in one place. [Here's one explanation.](https://jeffreyeverhart.com/2020/09/28/javascript-tips-using-named-constants-for-magic-numbers/) 

One example is what we did here for VZE colors: https://github.com/cityofaustin/atd-vz-data/blob/master/atd-vze/src/styles/colors.js.

Reply:
@mateoclarke Should I go in and add this value to a constants file?

Reply:
@jgabitto if you haven't talked about this with @mateoclarke already, go ahead and create an issue in atd-data-tech repo and use the `Type: Tech Debt` label. Remind us about it at next sprint planning and we'll queue it up for you.